okhttp/third_party/okhttp/java/io/grpc/okhttp/internal/Platform.java

```diff
@@ -105,47 +119,71 @@ public void connectSocket(Socket socket, InetSocketAddress address,
   }
 
   /** Attempt to match the host runtime to a capable Platform implementation. */
+  @SuppressWarnings("unchecked")
```

Comment:
This code is already pretty different from what we have in OkHttp. We’ve split these into their own files and added even more platform-specific APIs.

https://github.com/square/okhttp/blob/master/okhttp/src/main/java/okhttp3/internal/AndroidPlatform.java
https://github.com/square/okhttp/blob/master/okhttp/src/main/java/okhttp3/internal/Jdk9Platform.java

If you’d like to submit a PR to prefer Conscrypt on the JVM, I’m interested. My expectation is that the additional security and performance will be worth the effort!